src/components/FindWallet/WalletTable/index.tsx

```diff
@@ -0,0 +1,573 @@
+            bg: "primary",
+            color: "buttonColor",
+            _hover: {
+              bg: "primary",
+            },
+          },
+        },
+      },
+    }}
+    {...props}
+  />
+)
+
+const FlexInfo = (props: FlexProps) => (
+  <Flex
+    alignItems="center"
+    gap={4}
+    pl="0.3rem"
+    sx={{
+      p: {
+        p: 0,
+        fontSize: "1.2rem",
+        fontWeight: "bold",
+        "& + p": {
+          mt: "0.1rem",
+          fontSize: "0.9rem",
+          lineHeight: 4,
+          fontWeight: "normal",
+        },
+      },
+    }}
+    {...props}
+  />
+)
+
+const fadeIn = keyframes`
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+`
+
+const fadeOut = keyframes`
+  0% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0;
+  }
+`
+
+const FlexInfoCenter = (props: { children: ReactNode; className?: string }) => (
+  <FlexInfo
+    animation={`${fadeIn} 0.375s`}
+    cursor="pointer"
+    justifyContent="center"
+    height="full"
+    sx={{
+      "&.fade": {
+        animation: `${fadeOut} 0.375s`,
+      },
+    }}
+    {...props}
+  />
+)
+
+const SocialLink = (props: LinkProps) => (
+  <Link
+    display="flex"
+    height={8}
+    alignItems="center"
+    verticalAlign="middle"
+    transform="scale(1)"
+    transition="transform 0.1s"
+    _hover={{
+      transform: "scale(1.15)",
+    }}
+    {...props}
+  />
+)
+
+// Types
+export interface DropdownOption {
+  label: string
+  value: string
+  filterKey: string
+  category: string
+  icon: ReactNode
+}
+
+// Constants
+const firstCol = "firstCol"
+const secondCol = "secondCol"
+const thirdCol = "thirdCol"
+
+export interface WalletTableProps {
+  data: Record<string, any>
+  filters: Record<string, boolean>
+  walletData: WalletData[]
+}
+
+const WalletTable = ({ data, filters, walletData }: WalletTableProps) => {
+  const { t } = useTranslation()
+  const {
+    featureDropdownItems,
+    filteredFeatureDropdownItems,
+    filteredWallets,
+    setFirstFeatureSelect,
+    setSecondFeatureSelect,
+    setThirdFeatureSelect,
+    updateDropdown,
+    updateMoreInfo,
+    firstFeatureSelect,
+    secondFeatureSelect,
+    thirdFeatureSelect,
+    walletCardData,
+  } = useWalletTable({ filters, t, walletData })
+
+  return (
+    <Container>
+      <WalletContentHeader>
+        <Th>
+          {filteredWallets.length === walletCardData.length ? (
+            <Text as="span">
+              {t("page-find-wallet-showing-all-wallets")} (
+              <strong>{walletCardData.length}</strong>)
+            </Text>
+          ) : (
+            <Text as="span">
+              {t("page-find-wallet-showing")}{" "}
+              <strong>
+                {filteredWallets.length} / {walletCardData.length}
+              </strong>{" "}
+              {t("page-find-wallet-wallets")}
+            </Text>
+          )}
+        </Th>
+        <Th>
+          <Text as="span" hideFrom="sm" fontSize="md" whiteSpace="nowrap">
+            Choose features
```

Comment:
@pettinarip this only shows at the smaller mobile sizes and hidden under a [pseudo style](https://github.com/ethereum/ethereum-org-website/blob/a07e83d3baa50ff0209e57cc6b4fcfe89dc03d55/src/components/FindWallet/WalletTable.tsx#L124). So a translation would be needed for it since I now have it in the DOM

Reply:
Ah gotcha, thanks. I've pushed a new commit for that.